apps/console/src/extensions/identity-provider-templates/templates/swe/wizards/swe-authentication-provider-create-wizard.tsx

```diff
@@ -247,6 +245,12 @@ export const SIWEAuthenticationProviderCreateWizard: FunctionComponent<
             identityProvider.description = template.description;
             identityProvider.templateId = template.templateId;
 
+            identityProvider.federatedAuthenticators.authenticators[ 0 ].properties =
+                identityProvider?.federatedAuthenticators?.authenticators[ 0 ]?.properties?.filter(
+                    (item: CommonPluggableComponentPropertyInterface) =>
+                        item.key !== "ClientId" && item.key !== "ClientSecret" && item.key !== "callbackUrl"
```

Comment:
Shall we move these to constants?

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/6047